packages/modules/devices/shelly/config.py

```diff
@@ -0,0 +1,37 @@
+from modules.common.component_setup import ComponentSetup
+from helpermodules.auto_str import auto_str
+
+
+@auto_str
+class ShellyConfiguration:
+    def __init__(self, address: str = None):
+        self.address = address
```

Comment:
```suggestion
    def __init__(self, address: str = None, generation: Optional[int] = None):
        self.address = address
        self. generation = generation
```
generation kann mit in die Konfiguration aufgenommen werden. Es wird in 2.0 im UI nicht angezeigt, sondern im Device detektiert und dann gesetzt.

Reply:
gemacht